### PR TITLE
[15.0][ADD] stock_request_separate_picking

### PR DESCRIPTION
**Pull request by ps-tubtim**
This module automatically creates a new procurement group when creating a stock request/stock request order. Therefore one stock request/stock request order creates one new picking.

**Settings**
Selection_196

### CODE REVIEW

File: stock_request_separate_pickings/models/stock_request_abstract.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    _inherit = "stock.request.abstract"
+
+    procurement_group_id = fields.Many2one(
+        copy=False,
+    )
+
+    @api.model
+    def create(self, vals):
+        res = super().create(vals)
+        if not res.procurement_group_id:
```
Comment (rousseldenis) [inline stock_request_separate_pickings/models/stock_request_abstract.py]: @ps-tubtim As this break stock_request behavior, maybe a parameter to activate it (on company with res.config.settings) should be great

**Comment by pedrobaeza** [comment]
Please don't use plurals in module name.

**Review by PatthamaCH — APPROVED** [review]
ok

File: stock_request_separate_pickings/models/stock_request_abstract.py (hunk 1 above)
Comment (ps-tubtim) [reply stock_request_separate_pickings/models/stock_request_abstract.py]: @rousseldenis Thank you for your suggestion. I already add the **Allow Separate Picking** setting. Could you review this module again?

**Comment by ps-tubtim** [comment]
> Please don't use plurals in module name.

@pedrobaeza I already renamed this module to **stock_request_separate_picking**.  What do you think?

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**